test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -10,20 +10,28 @@ define([
 ], (BulkImportCell, Jupyter, Runtime, Mocks, TestUtil, TestAppObj, UI, TestAppSpec) => {
     'use strict';
     const fakeInputs = {
-        dataType: {
-            files: ['some_file'],
-            appId: 'someApp',
+            dataType: {
+                files: ['some_file'],
+                appId: 'someApp',
+            },
+        },
+        fakeSpecs = {
+            someApp: TestAppSpec,
         },
-    };
-    const fakeSpecs = {
-        someApp: TestAppSpec,
-    };
+        runtime = Runtime.make();
 
     describe('test the bulk import cell module', () => {
         beforeAll(() => {
             Jupyter.narrative = {
                 getAuthToken: () => 'fakeToken',
             };
+            jasmine.Ajax.install();
+            jasmine.Ajax.stubRequest(runtime.config('services.workspace.url')).andReturn({
+                status: 200,
+                statusText: 'HTTP/1.1 200 OK',
+                contentType: 'application/json',
+                responseText: '',
+            });
```

Comment:
the tests were making calls to the workspace, so this nips that little issue in the bud!